.github/pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+<!-- Provide a general summary of your changes in the Title above -->
+
+### Description
+<!--
+Describe your changes in detail.
+For contribution process guide, look into CONTRIBUTING.md
+
+If this is a relatively large or complex change:
+ - BEFORE creating a PR, find existing or start a new discussion on Issues tab,
+ - if the discussion is concluded, go ahead with this PR,
+ - perhaps describe what alternatives you considered.
+
+If this PR references or fixes an open issue, please link it here
```

Comment:
what about setting labels? bug or enhancement?

Reply:
with our current labels, I'm not really a fan of that. Maybe if we have labels for functionalities (e.g. like we had `pmem2` or `pmemobj`) that would make more sense for me...

If you think otherwise, I can add a sentence about it - please propose something :)